dev-util/stripe-mock/stripe-mock-0.34.1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGO_SRC="github.com/stripe/stripe-mock"
+EGO_PN=${EGO_SRC}/...
+inherit golang-build golang-vcs-snapshot
+
+DESCRIPTION="Mock HTTP server that responds like the real Stripe API"
+HOMEPAGE="https://github.com/stripe/stripe-mock"
+SRC_URI="https://github.com/stripe/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
+
+src_compile() {
+	GOPATH="${WORKDIR}/${P}:$(get_golibdir_gopath)" \
+		go build -v -work -x ${EGO_BUILD_FLAGS} "${EGO_PN}" -o stripe-mock
```

Comment:
Don't you need to `||die`?

Reply:
It turned out that the build command is wrong. I am now using the default command provided by `golang-build.eclass`

Reply:
Yes, good find!